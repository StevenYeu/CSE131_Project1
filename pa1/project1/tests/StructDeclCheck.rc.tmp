Error, "project1/tests/StructDeclCheck.rc": 
  Field x declared second time in struct.
Error, "project1/tests/StructDeclCheck.rc": 
  Field x declared second time in struct.
Error, "project1/tests/StructDeclCheck.rc": 
  undeclared identifier 'MYS'.
Error, "project1/tests/StructDeclCheck.rc": 
  Field y declared second time in struct.
Error, "project1/tests/StructDeclCheck.rc": 
  Duplicate declaration of overloaded function foo.
Error, "project1/tests/StructDeclCheck.rc": 
  Field bar declared second time in struct.
Error, "project1/tests/StructDeclCheck.rc": 
  Field bar declared second time in struct.
Error, "project1/tests/StructDeclCheck.rc": 
  Field bar declared second time in struct.
Compile: failure.
