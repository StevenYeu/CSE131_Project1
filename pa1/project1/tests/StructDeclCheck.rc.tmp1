  Field x declared second time in struct.
  Field x declared second time in struct.
  undeclared identifier 'MYS'.
  Field y declared second time in struct.
  Duplicate declaration of overloaded function foo.
  Field bar declared second time in struct.
  Field bar declared second time in struct.
  Field bar declared second time in struct.
Compile: failure.
