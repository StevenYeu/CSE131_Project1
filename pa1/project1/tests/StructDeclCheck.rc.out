Error, "/tmp/cs131_testrunner_GQ0qMF/file.rc": 
  Field x declared second time in struct.
Error, "/tmp/cs131_testrunner_GQ0qMF/file.rc": 
  Field x declared second time in struct.
Error, "/tmp/cs131_testrunner_GQ0qMF/file.rc": 
  undeclared identifier 'MYS'.
Error, "/tmp/cs131_testrunner_GQ0qMF/file.rc": 
  Field y declared second time in struct.
Error, "/tmp/cs131_testrunner_GQ0qMF/file.rc": 
  Duplicate declaration of overloaded function foo.
Error, "/tmp/cs131_testrunner_GQ0qMF/file.rc": 
  Field bar declared second time in struct.
Error, "/tmp/cs131_testrunner_GQ0qMF/file.rc": 
  Field bar declared second time in struct.
Error, "/tmp/cs131_testrunner_GQ0qMF/file.rc": 
  Field bar declared second time in struct.
Compile: failure.
